cmd/crl-checker/main.go

```diff
@@ -75,7 +77,15 @@ func main() {
 		}
 		if *emitRevoked {
 			for _, c := range crl.RevokedCertificates {
-				fmt.Printf("%x\n", c.SerialNumber)
+				serial := core.SerialToString(c.SerialNumber)
+				fmt.Println(serial)
+
+				if _, seen := seenSerials[serial]; seen {
+					errCount += 1
+					logger.Errf("Serial %s seen in multiple shards", serial)
```

Comment:
This only gets triggered if `*emitRevoked` is true. I think we should switch that around: always check  for duplicates, and print the serials only if `*emitRevoked`.